backend/server.js

```diff
@@ -69,7 +69,6 @@ app.get("/players", async (req, res) => {
       res.json(players);
```

Comment:
<img width="1716" alt="Screenshot 2024-07-24 at 8 01 54 PM" src="https://github.com/user-attachments/assets/6ffcf8f2-452f-424d-835b-329f185c3843">
Backend server is able to print out the exception without crashing, but because we don't send a response back, the frontend waits indefinitely. Will revisit tomorrow on how to get the frontend to recognize the exception faster.